cypress/deployed-and-local/integration/advancedSearch/search.cy.ts

```diff
@@ -104,10 +105,11 @@ describe('Advanced Search', () => {
       cy.get('[data-testid="case-detail-menu-button"]').click();
       cy.get('[data-testid="menu-button-upload-pdf"]').click();
       cy.get('[data-testid="upload-description"]').type(opinionTitle);
-      cy.get('[data-testid="primary-document-file"]').attachFile(
-        '../../helpers/file/sample.pdf',
-      );
-      cy.get('[data-testid="upload-file-success"]').should('exist');
+      attachFile({
+        filePath: '../../helpers/file/sample.pdf',
+        selector: '[data-testid="primary-document-file"]',
+        selectorToAwaitOnSuccess: '[data-testid^="upload-file-success"]',
+      });
```

Comment:
Because validating on file selection introduces a slight delay and causes tests to fail, I created this helper function to ensure our Cypress tests wait for success appropriately. A good amount of the git diff in this PR is just refactoring cy.get().attachFile to use this helper and renaming `uploadFile` to `attachSamplePdfFile` to avoid confusion with `attachFile`.